Makefile

```diff
@@ -58,6 +58,10 @@ init_data:
 	@echo "Aggregating data"
 	PYTHONPATH=.. python data/aggregator.py daily
 
+recommendations: require_virtualenv_in_dev
```

Comment:
We probably want to run this as a cron job, every week perhaps? We shouldn't need to do it for all our users, I'd say everyone who's logged in within the past week should suffice. Though maybe run it for everyone the first time.